src/components/City/SensorsQualityStats/DataTable.js

```diff
@@ -142,9 +139,7 @@ HumidityValue.propTypes = {
 function TemperatureValue({ classes, value }) {
   return (
     <DataValue value={value} classes={classes}>
-      <small className={classes.small}>
-        <sup>o</sup>C
-      </small>
+      <small className={classes.small}>&#8451;</small>
```

Comment:
Putting `°C` directly would have been better since we're already using `charset="utf-8"` in the html file.